### PR TITLE
New: UploadsManager drag and drop upload empty state

### PR DESCRIPTION
**Pull request by wenboyu2**
- adds an `isUploadsManagerVisible` prop to the `ContentUploader` to allow the UploadsManager to be shown when there's no ongoing uploads
- adds an empty state to the UploadsManager which instructs user to drop file to upload

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @wenboyu2 has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #435

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentUploader/ContentUploader.js [hunk 1]
```diff
@@ -233,57 +236,64 @@ class ContentUploader extends Component<Props, State> {
      * @param {Array<UploadFileWithAPIOptions | File>} files
      */
     getNewFiles = (files: Array<UploadFileWithAPIOptions | File>) => {
+        const { rootFolderId } = this.props;
         const { itemIds } = this.state;
 
-        return Array.from(files).filter((file) => !(this.getFileId(file) in itemIds));
+        return Array.from(files).filter((file) => !(getFileId(file, rootFolderId) in itemIds));
```
Comment (priyajeet) [inline src/components/ContentUploader/ContentUploader.js]: Not sure using "in" is the best. Isn't it outdated?
Better is to use array.includes

File: src/components/ContentUploader/ContentUploader.js [hunk 2]
```diff
@@ -233,57 +236,64 @@ class ContentUploader extends Component<Props, State> {
...
+        });
+
         clearTimeout(this.resetItemsTimeout);
 
-        const firstFile = getFile(files[0]);
+        const firstFile = getFile(newFiles[0]);
 
-        if (firstFile.webkitRelativePath) {
-            this.addFilesWithRelativePathToQueue(files, itemUpdateCallback);
+        if (firstFile.webkitRelativePath && !isRelativePathIgnored) {
```
Comment (tonyjin) [inline src/components/ContentUploader/ContentUploader.js]: Can you add some comments here? It's not clear to me why we only add the first item in this situation

Comment (wenboyu2) [reply src/components/ContentUploader/ContentUploader.js]: Thanks for you offline feedback, I updated the FolderUpload class interface which now only represents one folder.

File: src/components/ContentUploader/ContentUploader.js (hunk 1 above)
Comment (wenboyu2) [reply src/components/ContentUploader/ContentUploader.js]: `itemIds` is an object `{ [string]: boolean }`
I changed this to be just an object lookup instead, `!itemIds[getFileId(file, rootFolderId)]`

**Pull request merged**